### PR TITLE
Panel widget tweaks

### PR DESCRIPTION
**Pull request by aronhelser**
Several minor widget changes. 
Added an example for PropertyPanel, as I'm using it elsewhere. 

@jourdain CollapsibleWidget was setting a font-family, which caused consistency problems in my widget. Removing it might affect the appearance of other places that use it. Anything we need to check?

### CODE REVIEW

**Comment by jourdain** [comment]
@TristanWright can you review

File: src/React/Widgets/NumberSliderWidget/index.js [hunk 1]
```diff
@@ -62,14 +62,14 @@ export default React.createClass({
...
-          max={max} min={min}
+          max={max} min={min} step={this.props.step || 'any'}
         />
         <input
           type="number"
           className={style.text}
           value={this.props.value}
           onChange={this.valInput}
-          max={max} min={min}
+          max={max} min={min} step="any"
```
Comment (TristanWright) [inline src/React/Widgets/NumberSliderWidget/index.js]: Neat, I didn't know about the 'any' option for `<input step>`. Shouldn't it be the same as the slider though? `this.props.step || 'any'`?

File: src/React/Properties/PropertyPanel/index.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ export default React.createClass({
 
   render() {
     var viewData = this.props.viewData,
-      uiContents = content => factory(content, viewData, this.valueChange),
+      uiContents = content => factory(content, viewData, this.props.onChange ? this.valueChange : undefined),
```
Comment (TristanWright) [inline src/React/Properties/PropertyPanel/index.js]: Does this make the block in `valueChange`

```
if (this.props.onChange) {
  this.props.onChange(newVal)
}
```

above redundant?

File: src/React/Widgets/NumberSliderWidget/index.js (hunk 1 above)
Comment (aronhelser) [inline src/React/Widgets/NumberSliderWidget/index.js]: I should test it in IE :) 
I didn't want the number input to turn red and complain if you enter a more precise number. However, it does affect what the arrows inside the text input do. I wish they could be controlled separately.

File: src/React/Properties/PropertyPanel/index.js (hunk 2 above)
Comment (TristanWright) [inline src/React/Properties/PropertyPanel/index.js]: Alright thanks for clearing that up

File: src/React/Widgets/NumberSliderWidget/index.js (hunk 1 above)
Comment (TristanWright) [inline src/React/Widgets/NumberSliderWidget/index.js]: Got it. LGTM

**Pull request merged**